ansible/pbTestScripts/buildJDK.sh

```diff
@@ -10,6 +10,7 @@ export JAVA_TO_BUILD=jdk8u
 export VARIANT=openj9
 export JDK7_BOOT_DIR=/usr/lib/jvm/java-1.7.0
 export JDK_BOOT_DIR=/usr/lib/jvm/java-8-openjdk-amd64
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
```

Comment:
different URL to the JDK_BOOT_DIR - aren't they the same VM?

Reply:
Weirdly enough, both are there in an Ubuntu1604 VM, however, they are the exact same build. Not sure why both are there to begin with to be honest.